packages/react/src/lib/accessibility/Behaviors/Button/splitButtonBehavior.ts

```diff
@@ -0,0 +1,66 @@
+import { Accessibility } from '../../types'
+import * as keyboardKey from 'keyboard-key'
+import * as _ from 'lodash'
+import menuButtonBehavior from '../MenuButton/menuButtonBehavior'
+
+/**
+ * @specification
+ * Adds role='button' if element type is other than 'button'. This allows screen readers to handle the component as a button.
+ * Adds attribute 'tabIndex=0' if element type is other than 'button'.
+ * Adds attribute 'aria-disabled=true' based on the property 'disabled'. This can be overriden by providing 'aria-disabled' property directly to the component.
+ * Triggers 'performClick' action with 'Enter' or 'Spacebar' on 'root'.
+ */
+const buttonBehavior: Accessibility<SplitButtonProps> = props => {
+  const splitButtonMenuButtonBehavior = props => {
+    const menuButtonBehaviorData = menuButtonBehavior(props)
+
+    return _.merge(menuButtonBehaviorData, {
+      attributes: {
+        trigger: {
+          tabIndex: -1,
+        },
+      },
+      keyActions: {
+        popup: {
+          closeAndFocusTrigger: {
+            keyCombinations: [
+              { keyCode: keyboardKey.Escape },
+              { keyCode: keyboardKey.ArrowUp, altKey: true },
+            ],
+          },
+        },
+      },
+    })
+  }
+  return {
+    attributes: {
+      root: {
+        tabIndex: 0,
```

Comment:
this will make the span focusable, but the onClick is on the button - it will not work correctly with VO keys.

Reply:
addressed